README.md

```diff
@@ -172,8 +172,10 @@ onto a Docker container and used to create a virtual environment capable of runn
 The plugin will automatically inject logic for building and using this virtual environment into a pre-existing Dockerfile
 by default. The `dockerfile` configuration must be set to the target Dockerfile. To disable the Dockerfile update
 altogether, set the `updateDockerfile` configuration to `false`. Because the final image must match the image that builds
-the virtual environment, Habushu must know what the base image for the final output should be. By default, `python:3.11-slim`
-is used, but this can be customized with the `dockerBase` and `dockerUser` configurations.
+the virtual environment, Habushu defaults to using `python:3.11` to build the virtual environment and  `python:3.11-slim`
+as the final image that packages/runs the virtual environment. This can be customized with the `dockerBuilderBase`,
+`dockerFinalBase`, and `dockerUser` configurations, but care must be taken to ensure the builder image platform is
+compatible with the final image platform so that the virtual environment is compatible.
```

Comment:
A: doesn't have to be exactly this, but the current working just felt like it was using the word compatible too much
```suggestion
`dockerFinalBase`, and `dockerUser` configurations, but care must be taken to ensure the virtual environment is compatible with both the builder image and final image platforms.
```

Reply:
Ah I didn't even notice I used the word twice. I'll clean it up.